### PR TITLE
Use the correct cluster policy for upgrades

### PR DESCRIPTION
**Pull request by michalskrivanek**
InClusterUpgrade is wrong, there is a new policy introduced in oVirt 4.1.4

### CODE REVIEW

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by michalskrivanek** [comment]
not sure how it behaves in <4.1.4, but there is not useful policy there anyway

File: roles/ovirt-cluster-upgrade/library/ovirt_scheduling_policies_facts.py [hunk 1]
```diff
@@ -50,9 +50,9 @@
 # Examples don't contain auth parameter for simplicity,
 # look at ovirt_auth module to see how to reuse authentication:
 
-# Gather facts about all scheduling policies with name InClusterUpgrade:
+# Gather facts about all scheduling policies with name ClusterInMaintenance:
 - ovirt_scheduling_policies_facts:
-    name: InClusterUpgrade
+    name: ClusterInMaintenance
```
Comment (mwperina) [inline roles/ovirt-cluster-upgrade/library/ovirt_scheduling_policies_facts.py:55]: So do we need to add logic here to use InClusterUpgrade for oVirt < 4.1.4 and ClusterInMaintenance for oVirt >= 4.1.4?

Comment (michalskrivanek) [reply roles/ovirt-cluster-upgrade/library/ovirt_scheduling_policies_facts.py:55]: no, the InClusterUpgrade is completely useless in oVirt 4.0+

Comment (machacekondra) [reply roles/ovirt-cluster-upgrade/library/ovirt_scheduling_policies_facts.py:55]: But `ClusterInMaintenance` is only in 4.1.4, so what should we use prior to that version? We can't use `ClusterInMaintenance` in <4.1.4.

Comment (michalskrivanek) [reply roles/ovirt-cluster-upgrade/library/ovirt_scheduling_policies_facts.py:55]: in <4.1.4 it should not use/change any policy. As long as the code skips over those change/restore policy parts it's good enough

**Comment by machacekondra** [comment]
Closing this and continue in https://github.com/oVirt/ovirt-ansible/pull/34

**Pull request closed without merging**